docs/platform/howto/disable-platform-authentication.rst

```diff
@@ -0,0 +1,18 @@
+Disable platform authentication
+===============================
+
+If you have already :doc:`set up a SAML authentication <saml/saml-authentication>` method then you might want to disable the default password-based platform authentication method to prevent it being used to access projects within your account.
+
+Here are the steps to follow to disable platform authentication:
+
+1. Log in to the Aiven Console using your alternative authentication (for example, SAML) method. You must not be logged in with any open sessions using the password-based platform authentication.
```

Comment:
Should the `You must not be logged in with any open sessions using the password-based platform authentication.` be a warning?